src/main/io/ledstrip.c

```diff
@@ -646,11 +646,37 @@ static void applyLedVtxLayer(bool updateNow, timeUs_t *timer)
     }
     else { // show frequency
         // calculate the VTX color based on frequency
-        int hue = constrain((frequency - 5645.0 ) * 1.2, 0, 360);
-        // if we ever want to wrap the hue around the wheel for L band frequencies...
-        //hue = (hue+(hue<0)*((0-hue)/360+1)*361)%361;
-        color.h = hue;
-        color.s = 0;
+	    color.s = 0;
+	    int channel = constrain(floor((Frequency-5633)/39)+1, 1, 8);
```

Comment:
You don't need to use `floor`. `int` calculations will automatically be rounded down.